ArcGISRuntimeSDKQt_QMLSamples/Layers/ClassBreaksWithAlternateSymbols/ClassBreaksWithAlternateSymbols.qml

```diff
@@ -0,0 +1,172 @@
+// [Legal]
+// Copyright 2022 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import Esri.ArcGISRuntime 100.13
+import Esri.ArcGISExtras 1.1
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    MapView {
+        id: mapView
+        anchors.fill: parent
+
+        Map {
+            initBasemapStyle: Enums.BasemapStyleArcGISTopographic
+
+            // create the feature layer
+            FeatureLayer {
+                id: featureLayer
+
+                // feature table
+                ServiceFeatureTable {
+                    id: featureTable
+                    url: "https://sampleserver6.arcgisonline.com/arcgis/rest/services/SF311/FeatureServer/0"
+                }
+            }
+            onLoadStatusChanged: {
+
+                if (loadStatus === Enums.LoadStatusLoaded) {
+                    mapView.setViewpoint(initialViewpoint);
+                }
+
+                // create the class breaks renderer with alternate symbols and assign it to the feature layer renderer
+                featureLayer.renderer = createClassBreaksRenderer_();
+            }
+
+
+
+        }
+    }
+
+    ViewpointCenter {
+            id: initialViewpoint
+            center: Point {
+                x: -13632095.660131
+                y: 4545009.846004
+            }
+            targetScale: 50000
+        }
+
+    Column {
+        anchors {
+            left: parent.left
+            top: parent.top
+            margins: 15
+        }
+        spacing: 5
+
+        Button {
+            width: 200
+            enabled: true
+            text: "Set Scale to 2500"
+            onClicked:
+            {
+                const scale = 2500;
+                mapView.setViewpointCenterAndScale(ArcGISRuntimeEnvironment.createObject("Point", {x: -13632095.660131, y: 4545009.846004}), scale);
+            }
+        }
+
+        Button {
+            width: 200
+            enabled: true
+            text: "Set Scale to 7500"
+            onClicked:
+            {
+                const scale = 7500;
+                mapView.setViewpointCenterAndScale(ArcGISRuntimeEnvironment.createObject("Point", {x: -13632095.660131, y: 4545009.846004}), scale);
+            }
+        }
+
+        Button {
+            width: 200
+            enabled: true
+            text: "Set Scale to 15000"
+            onClicked:
+            {
```

Comment:
QML/Javascript differs from C++ in that brackets should be placed on the previous line rather than their own
```suggestion
            onClicked: {
```